package.json

```diff
@@ -26,7 +26,7 @@
   },
   "peerDependencies": {
     "chai": "3.x",
-    "cheerio": "0.19.x || 0.20.x",
+    "cheerio": "0.22.0",
```

Comment:
Breaks support for existing peer deps. So 👎 for this incarnation. Think adding `|| 0.22.x" would be ideal.